app/src/main/java/org/p2p/wallet/utils/KeyExtensions.kt

```diff
@@ -26,3 +26,9 @@ fun String.cutEnd(): String {
     val lastFour = this.takeLast(4)
     return "$firstSixteen...$lastFour"
 }
+
+@Suppress("MagicNumber")
+fun String.cutStart(): String {
```

Comment:
please provide int to cut some number of symbols

Reply:
you can use 4 as Default, and move it to so magic const x D

Reply:
kek